presto-main/src/test/java/com/facebook/presto/spiller/TestFileSingleStreamSpiller.java

```diff
@@ -79,13 +79,18 @@ public void testSpill()
         spiller.spill(Iterators.forArray(page, page, page)).get();
         assertEquals(listFiles(spillPath.toPath()).size(), 1);
 
-        // for spilling memory should be accounted only during spill() method is executing
-        assertEquals(memoryContext.getBytes(), 0);
+        // previously, for spilling memory was accounted only during spill() method is executing
+        // but, due to concurrency issues (spillers can run in different threads than the driver thread)
+        // the reclamation of the buffers are deferred until the spiller is closed
+        // assertEquals(memoryContext.getBytes(), 0);
```

Comment:
Please write this comment from the perspective of future readers. The way it's currently written, it's comparing with previous behavior, which is not of interest to future readers. It will also get harder to read as the code continues to evolve.